src/main/webapp/css/style.css

```diff
@@ -1,15 +1,9 @@
-body {
-  margin: 0;
-  padding: 0;
+#content {
+  margin-left: auto;
```

Comment:
Does `margin: auto` work here instead of these two lines?

Reply:
Not entirely sure, I pulled this from the master branch.